peps/pep-0774.rst

```diff
@@ -0,0 +1,255 @@
+PEP: 774
+Title: Removing the LLVM requirement for JIT builds
+Author: Savannah Ostrowski <savannah@python.org>
+Status: Draft
+Type: Standards Track
+Created: 26-Jan-2025
```

Comment:
We normally say "Created" is when the PEP number was assigned / PR opened (2025-01-27 at 03:06 GMT)
```suggestion
Created: 27-Jan-2025
```